documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -0,0 +1,517 @@
+[[getting-started]]
+= Introduction to Hibernate Reactive
+
+:example: https://github.com/hibernate/hibernate-reactive/tree/master/example
+
+Before you start, we recommend taking a quick look at the example
+program in the {example}[`example`] directory, which shows off all
+the "bits" you'll need to get your own program up and running.
+
+== Setting up a reactive Hibernate project
+
+Setting up a project that uses Hibernate Reactive isn't hard at all.
+
+If you're using Hibernate Reactive outside of the Quarkus environment,
+you'll need to:
+
+- include Hibernate Reactive itself along with the appropriate Vert.x
+  reactive database client as dependencies of your project, and
+- configure Hibernate Reactive with information about your database,
+  using Hibernate configuration properties.
```

Comment:
Maybe we can add the GAV coordinates of the dependencies for reactive and the drivers?

Reply:
I've just realized that they are mentioned in the later section

Reply:
Yes, and note I don't want to add version numbers to this document, because that would be a maintenance nightmare.

Remember, @DavideD, well-written technical docs should always have handwavey preambles at the beginning of sections, before getting down to nitty gritty technical details in titled subsections.